### DISCUSSION TITLE
How does it work?

### DISCUSSION THREAD

**Discussion by Rabtos** [tomasrudh/WinSize4#4] [General]
Is it bugged for me or why don't I see anything in the list?

**Comment by Peter-202122** [comment]
> [...] why don't I see anything in the list?

Because it is your decision what you want to have in the list. ;-) 

1. Make sure that WinSize4 is running.
2. Open the window of the app for which you would like a stable window position respectively window size.
3. Bring this window in the desired position respectively size on your screen.
4. Click into the window (or in its title bar).
5. Press the keys **Ctrl + Alt + Z** 
--> An appropriate entry will be created automatically in the list of WinSize4.
